compiler/modules/bir/verify.bal

```diff
@@ -151,6 +151,48 @@ function verifyInsn(VerifyContext vc, Insn insn) returns Error? {
     else if insn is ErrorConstructInsn {
         check validOperandString(vc, name, insn.operand, insn.pos);
     }
+    else if insn is TypeMergeInsn {
+        check verifyTypeMerge(vc, insn);
+    }
+    else if insn is TypeBranchInsn {
+        check verifyTypeBranch(vc, insn);
+    }
+}
+
+function verifyTypeMerge(VerifyContext vc, TypeMergeInsn insn) returns err:Internal? {
+    Register opRoot = unnarrow(insn.result);
```

Comment:
`opRoot` also need to change now. How about unnarrowedOp

Reply:
Changed.